build.gradle

```diff
@@ -166,6 +166,7 @@ dependencies {
     compile group: 'net.sourceforge.argparse4j', name: 'argparse4j', version: '0.7.0'
     compile group: 'net.sourceforge.pcgen', name: 'PCGen-base', version:'1.0.55'
     compile group: 'net.sourceforge.pcgen', name: 'PCGen-Formula', version:'1.0.64'
+    compile group: 'org.jetbrains', name: 'annotations', version:'15.0'
```

Comment:
It serves as a hinter for FindBugs, Eclipse/IntelliJ, and various other tools.
See for example http://help.eclipse.org/kepler/index.jsp?topic=/org.eclipse.jdt.doc.user/tasks/task-using_null_annotations.htm